flowauth/backend/tests/test_token_admin.py

```diff
@@ -13,3 +16,20 @@ def test_list_tokens_for_server(client, auth, test_admin):
     response = client.get("/admin/tokens", headers={"X-CSRF-Token": csrf_cookie})
     assert 200 == response.status_code
     assert 1 == len(response.get_json())
+
+
+@pytest.mark.usefixtures("test_data")
+def test_get_server_public_key(app, client, auth):
+    """Test that admin can get the flowauth server's public key"""
+    response, csrf_cookie = auth.login("TEST_ADMIN", "DUMMY_PASSWORD")
+
+    response = client.get("/admin/public_key", headers={"X-CSRF-Token": csrf_cookie})
+    key_json = response.get_json()
+    expected_key = load_public_key(environ["PUBLIC_JWT_SIGNING_KEY"]).public_bytes(
```

Comment:
Since we use the `public_bytes(
        encoding=serialization.Encoding.PEM,
        format=serialization.PublicFormat.SubjectPublicKeyInfo,
    )` mantra in a bunch of places, is it worth pushing this into the definition of `load_public_key` itself? (Or are we making use of the return value of `load_public_key` in different ways elsewhere?)